editorconfig_test.go

```diff
@@ -95,6 +85,31 @@ func TestGetDefinition(t *testing.T) {
 	assert.Equal(t, EndOfLineLf, def.EndOfLine)
 }
 
+func TestWrite(t *testing.T) {
+	ec, err := ParseFile(testFile)
+	if err != nil {
+		t.Errorf("Couldn't parse file: %v", err)
+	}
+
+	tempFile := filepath.Join(os.TempDir(), ".editorconfig")
+	defer os.Remove(tempFile)
+
+	f, err := os.OpenFile(tempFile, os.O_CREATE|os.O_TRUNC|os.O_WRONLY, 0644)
+	assert.Nil(t, err)
+	defer os.Remove(tempFile)
+
+	err = ec.Write(f)
+	assert.Nil(t, err)
+
+	f.Close()
+
+	savedEc, err := ParseFile(tempFile)
+	assert.Nil(t, err)
+
+	testParse(t, savedEc)
+	os.Remove(tempFile)
```

Comment:
This statement is duplicated and can be removed

Reply:
@andreynering thanks for the review, I've made the changes.

Reply:
Nice! Feel free to merge once you're done. 😉